### ISSUE TITLE
USWDS - Bug: Combo Box clear input button focus outline does not match assigned value

### ISSUE THREAD

**Issue by DanielKing-1** [uswds/uswds#6424]
### Describe the bug

Currently, the combo box clear input button applies a lighter focus outline color than what is assigned. For example, when looking at the USWDS component demo and applying focus to the button, the _focus.scss looks like it should be applying an outline color of #2491ff, but in reality, #7CBDFF is being assigned.

When accepting the default outline colors, this fails WCAG 2.1 Non-text Contrast (AA) as the contrast ratio is 2:1.

### Steps to reproduce the bug

1. Go to the [Combo Box preview](https://designsystem.digital.gov/components/combo-box/#component-preview)
2. Apply focus on the Clear Input button
3. Compare assigned focus outline color to displayed color

![Image](https://github.com/user-attachments/assets/aa15be40-10df-4d5f-8d39-6ed2dcb0bdee)

### Expected Behavior

Expected behavior is that the assigned outline color for focus is applied to the Clear Input button.

### System setup

- This is experienced with both Chrome 133.0.6943.127 and Firefox 135.0.1

### Additional context

To get around the 508 issue, a darker color can be assigned, however the assigned color is still not the same as the displayed color.

![Image](https://github.com/user-attachments/assets/97d38574-a256-4d53-92a8-ef6c97131c5d)

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by amycole501** [comment]
I can confirm that the focus styles intended for the clear button's focus state DOES pass (2491ff)  but the color that is applied, #7CBDFF, doesn't.
